doc/troubleshooting/howtoset_sel.md

```diff
@@ -0,0 +1,15 @@
+# How to set sel ?
+
+The setting of "sel" is related to "rcut" and the coordination number of certain atoms. Some empirical settings on some specific systems are listed below.
```

Comment:
This explanation is not clear to users.  Explicitly write down the definition of `sel` and the principle of setting `sel`

Reply:
Done.